source/_posts/2019-03-11-upgrading-to-embark-4.md

```diff
@@ -1,7 +1,9 @@
 title: How to upgrade to Embark 4
 summary: "In this guide, we'll learn how to upgrade a Dapp created with Embark 3.x to Embark 4"
+author: jonathan_rainville
 categories:
   - tutorials
+layout: blog-post
```

Comment:
With the new theme we now have `author` and a dedicated blog post layout. The layout includes publisher information in the header.